### ISSUE TITLE
Quest Issue

### ISSUE THREAD

**Issue by Mehchanic** [elysium-project/classic-bug-tracker#903]
Seed of Life in Moonglade does not complete with Seed of life in inventory

**Comment by Thenari** [comment]
https://classicdb.ch/?quest=7066
I assume you mean this quest

**Comment by Thenari** [comment]
So you got the seed after killing princess in mara then you go to moonglade and talk to remulus with the seed. Gonna need at least a screenshot that its not working.

**Comment by Mehchanic** [comment]
Apologies for late reply. 
![image](https://user-images.githubusercontent.com/39348072/46842375-1cf41180-cd82-11e8-9a77-6f0b7a2f7890.png)
I havent been able to turn it in.

**Comment by Thenari** [comment]
Yeah i do remember a lot of tickets like this on earlier patches. They were mostly solved by telling a GM and the GM would remove the quest from you and give it back. Doing so does sometimes(its not consistant) delete quest items.

This character isnt happen to be a faction transfer or any history with the elysium web store? Those transfers sometimes have residue for example if that seed of life was on the previous owner and after transfer trying to hand in. Might mess with stuff

**Comment by Thenari** [comment]
Theres also always the option to delete the item and quest cache in that folder before you launch the client might be you havent updated that in awhile.

Either WTF or WDB i get them confused

**Comment by Mehchanic** [comment]
No, this character hasnt undergone faction transfer. I'll try deleting the cache.

**Comment by Mehchanic** [comment]
After deleting cache I still get the error.
